### ISSUE TITLE
Add "how to contribute" section to the textbook 

### ISSUE THREAD

**Issue by HuangJunye** [qiskit-community/qiskit-textbook#688]
**Describe the problem**
It's not obvious to new readers that the textbook is open source and they can make contributions to make it better. 

**Describe the expected solution**
I think it would be great to add a section of "how to contribute" similar to Qiskit Documentation. It might be a good idea to add something like "Correct me on Github"" ribbon similar to "Fork me on Github" on every page.

**Comment by rochisha0** [comment]
I would like to write the section on "how to contribute" for qiskit textbook @HuangJunye !

**Comment by HuangJunye** [comment]
@rochisha0 Thanks! Let’s see if @frankharkins  think it makes sense to add this section.

**Comment by frankharkins** [comment]
Where do you see this section fitting in? In the prerequisites? And where would the ribbon go?

**Comment by HuangJunye** [comment]
Yes, I think it fits well in the prerequisites. I just noticed there is "Contribute on GitHub" link on each chapter on the side panel under "Download as Jupyter Notebook". I guess we can make it a bit more obvious? Not sure what's the best way to do that.

**Comment by HuangJunye** [comment]
"Contribute on GitHub" points to the corresponding chapter on GitHub. This is not the correct workflow. It's not likely that readers will change the notebook directly on GitHub website. I think it should direct to the "How to contribute" and from there readers see 1) how to raise an issue if they want to suggest changes, 2) how to setup on their computer if they want to fix the issues themselves.

**Comment by Aniruddha120** [comment]
Hello everyone! It is really a good decision to add that feature. But I would like to share my experience with the webpage of the textbook. Previously, I wanted to add a feature that was ""night mode" and tried to do so(hoping you would like it after pulling a request of it). But, the website seemed too complicated to me.. maybe I am not that much skilled, but I think I should put my findings of it.. If it is also complicated to you, then I think you may also find difficulty to add""download as notebook" feature to it also.. Thank you and expecting your kind comments regarding it.

**Comment by rochisha0** [comment]
@HuangJunye you are right! That link 'Contribute on GitHub' should point to the 'How to contribute' section or the issues page on GitHub. 
Though I think the section should be added in appendix rather than prerequisites, because it is not necessary to know while using the book. Let me know what do you think.

**Comment by HuangJunye** [comment]
Yes I think Appendix makes more sense. I was thinking adding prerequisite will make this section more visible, but i guess it doesn’t matter if “Contribute in Github” directs to this section.

**Comment by frankharkins** [comment]
I agree the section should go in the appendix. I think it is useful to see the notebook the page was built from, so maybe the sidebar should say:
- Download as Jupyter Notebook
- View Jupyter Notebook on Github (will link to page on github)
- Edit the Textbook (link to new "how to contribute" section)

Since it's relevant, there is also a [`CONTRIBUTING.md`](https://github.com/qiskit-community/qiskit-textbook/blob/master/CONTRIBUTING.md) file that is not currently very visible. Perhaps some of this can be useful?

**Comment by rochisha0** [comment]
Yes this is great. Perhaps to make the CONTRIBUTING.MD more visible we can add a Contribution section in README.md (which is present in most github repositories).

**Comment by frankharkins** [comment]
Thanks for your contribution. I'm closing this issue as this repository is being archived. You can find the latest version of the Qiskit Textbook online at <https://qiskit.org/learn/>, and the new content repository at [Qiskit/textbook](https://github.com/Qiskit/textbook). Please feel free to re-open this issue there if it's still relevant.
